contact_map/contact_trajectory.py

```diff
@@ -0,0 +1,368 @@
+from collections import abc, Counter
+
+from .contact_map import ContactFrequency, ContactObject
+import json
+
+class ContactTrajectory(ContactObject, abc.Sequence):
+    """Track all the contacts over a trajectory, frame-by-frame.
+
+    Internally, this has a single-frame :class:`.ContactFrequency` for each
+    frame of the trajectory.
+
+    Parameters
+    ----------
+    trajectory : mdtraj.Trajectory
+        the trajectory to calculate contacts for
+    query : list of int
+        Indices of the atoms to be included as query. Default ``None``
+        means all atoms.
```

Comment:
this is heavy atoms and not water (from `ContactObject`)
We might want to change that default
```suggestion
        means all heavy, non-water atoms.
```

Reply:
I think I took this as default because MDTraj had something similar. One thing: if the default atoms change, the default cutoff should also change.